src/components/ConnectButton.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button } from '@chakra-ui/react';
+import { Trans } from '@lingui/macro';
+import { ConnectKitButton } from 'connectkit';
+import type { FunctionComponent } from 'react';
+
+import { MetaMaskIcon, AvatarIcon, AvatarFallbackIcon } from './icons';
+
+export type CustomConnectButtonProps = {
+  handleOnClick?: (() => void) | undefined;
+  isConnected: boolean;
+  ensName?: string | undefined;
+  truncatedAddress?: string | undefined;
+};
+
+export const CustomConnectButton: FunctionComponent<
+  CustomConnectButtonProps
+> = ({ handleOnClick, isConnected, ensName, truncatedAddress }) => {
+  let icon = <MetaMaskIcon width={{ base: '0', sm: '1.3rem' }} />;
+  if (isConnected) {
+    if (ensName) {
+      icon = <AvatarIcon width={{ base: '0', sm: '1.3rem' }} />;
+    } else {
+      icon = <AvatarFallbackIcon width={{ base: '0', sm: '1.3rem' }} />;
+    }
+  }
```

Comment:
Maybe extract this into a separate component, and move the width into a `const`.

Reply:
agree i think thats better

Reply:
resolved